aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -117,7 +117,7 @@ export const TableCustomizationExample = () => {
                   actions={
                     <Box direction="row" align="start" gap="small" flex={false}>
                       <DropButton
-                        icon={<SettingsOption />}
+                        icon={<Splits />}
```

Comment:
We have other accessibility improvements we should do for once the drop opens, however, we should provide an a11yTitle here at a minimum.

```suggestion
                        a11yTitle="Configure columns, button"
                        icon={<Splits />}
```

Reply:
We should also include `aria-labeledby="search-cols-icon"` in the [columns visibility example](https://deploy-preview-2088--keen-mayer-a86c8b.netlify.app/templates/datatable-customization?q=data#column-visibility)

I'm okay if we create an issue for "Fix accessibility issues on DataTable Customization page" if you'd prefer for the search icon and datatable checkbox WAVE errors.